bot/cogs/teamdraft.py

```diff
@@ -132,25 +130,31 @@ async def _process_pick(self, reaction, user):
     async def draft(self):
         """ Start the team draft and return the teams after it's finished. """
         # Initialize draft
+        guild_data = await self.bot.db_helper.get_guild(self.guild)
         self.users_left = self.users.copy()  # Copy users to edit players remaining in the player pool
         self.teams = [[], []]
+        captain_method = guild_data['captain_method']
 
-        if CAPTAINS == 'high score':
+        if captain_method == 'rank':
             players = await self.bot.api_helper.get_players(self.users_left)
             players.sort(reverse=True, key=lambda x: x.score)
 
             for team in self.teams:
                 captain = self.bot.get_user(players.pop(0).discord)
                 self.users_left.remove(captain)
                 team.append(captain)
-        elif CAPTAINS == 'random':
-            rand_users = self.users_left.copy()  # Create new list to preserve original order
-            random.shuffle(rand_users)
+        elif captain_method == 'random':
+            temp_users = self.users_left.copy()
+            random.shuffle(temp_users)
```

Comment:
Would be nice to have an approval process for the random structure in the future

Reply:
Can you elaborate what you mean?

Reply:
Yeah, we could show what the randomly generated teams are and then process tick and cross reactions to see whether we should re-generate and re-approve. Perhaps set an approval threshold of like 80% or something like that to prevent heavily one-sided teams.

Reply:
Ah I see. Yeah this was a one I threw in to just have a quick way to make teams. Can look at this down the road perhaps.